### PR TITLE
Added _976.java

### PR DESCRIPTION
**Pull request by varunu28**

### CODE REVIEW

File: src/main/java/com/fishercoder/solutions/_976.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package com.fishercoder.solutions;
+
+import java.util.Arrays;
+
```
Comment (fishercoder1534) [inline src/main/java/com/fishercoder/solutions/_976.java]: Could you add problem descriptions as comments in each problem as well? That'll save time for future reference.

Thanks!

Comment (varunu28) [reply src/main/java/com/fishercoder/solutions/_976.java]: Done

Comment (fishercoder1534) [reply src/main/java/com/fishercoder/solutions/_976.java]: Nit: Problem title with problem # also? :)

Comment (varunu28) [reply src/main/java/com/fishercoder/solutions/_976.java]: Added

Comment (fishercoder1534) [reply src/main/java/com/fishercoder/solutions/_976.java]: 👍 Thanks!

**Pull request merged**